ui/public/scripts/components/mobile-app/view-app-config.component.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+/**
+ * @ngdoc component
+ * @name mcp.component:view-app-config
+ * @description
+ * # view-app-config
+ */
+angular.module('mobileControlPanelApp').component('viewAppConfig', {
+  template: `<div>
+              <div class="row app-config">
+                <div class="col-lg-6">
+                  <h3>Details</h3>
+                  <dl class="dl-horizontal left">
+                    <div>
+                      <dt>Name</dt>
```

Comment:
Could we use 'Jenkins Job Name' here, and move the input down the form? It's lower priority than the Repo details

Reply:
Have updated this here and on the other edit/create views